apps/src/redux.js

```diff
@@ -176,7 +176,8 @@ function createStore(reducer, initialState) {
               'runApp',
               'props.showNextHint',
               'props.assetUrl',
-              'props.exportApp'
+              'props.exportApp',
```

Comment:
as I was testing I found these additional redux violations, so I added them to the ignore list.